framework/src/mediaplayer/mplayer_debug.h

```diff
@@ -0,0 +1,47 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <stdio.h>
+#include <debug.h>
+
+#define DEBUG_NONE	0x0
```

Comment:
This block will cause name collision. DEBUG_WARN etc

Reply:
I think there will be no collision due to DEBUG_WARN and other definitions since the file in which they are defined cannot be included by application. Plus the scope of definition is limited to the files where the mplayer_debug.h file is included, so it would not affect other modules.